benches/bench.rs

```diff
@@ -360,6 +374,128 @@ fn bench_openssl(c: &mut Criterion) {
 #[cfg(not(feature = "tests-openssl"))]
 fn bench_openssl(_c: &mut Criterion) {}
 
+#[cfg(feature = "tests-glass-pumpkin")]
+fn bench_glass_pumpkin(c: &mut Criterion) {
+    use crypto_bigint::Limb;
+    use crypto_primes::hazmat::{lucas_test, AStarBase, LucasCheck, MillerRabin, Primality};
+
+    fn required_checks(bits: u32) -> usize {
+        ((bits as f64).log2() as usize) + 5
+    }
```

Comment:
This looks arbitrary and could use a motivating comment.

```suggestion
    // The `glass-pumpkin` implementation is doing a different number of M-R checks than this crate.
    // For a fair comparison we make a custom implementation here, using the same number of checks that `glass-pumpkin` does.
    fn required_checks(bits: u32) -> usize {
        ((bits as f64).log2() as usize) + 5
    }
```